cli/zone.go

```diff
@@ -82,19 +82,21 @@ func runGetZone(cmd *cobra.Command, args []string) {
 	}
 
 	db := makeSQLClient()
-	err = runQueryWithFormat(db, fmtMap{"config": formatZone}, `SELECT * FROM system.zones WHERE id=$1`, id)
+	_, rows, err := runQueryWithFormat(db, fmtMap{"config": formatZone},
+		`SELECT * FROM system.zones WHERE id=$1`, id)
 	if err != nil {
 		log.Error(err)
 		return
 	}
 
-	if err != nil {
-		log.Error(err)
+	if len(rows) == 0 {
+		fmt.Fprintf(osStdout, "Object %d: no zone config found\n", id)
 		return
 	}
+	fmt.Fprintf(osStdout, "Object %d:\n%s\n", id, rows[0][1])
```

Comment:
We shouldn't include the `Object %d` header here. The caller knows the ID since it was present on the command line and it just gets in the way of sequences like `zone get | sed | zone set`.